sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_events_v1/query.sql

```diff
@@ -1,18 +1,76 @@
-WITH _events_ping_distinct_client_count AS (
+WITH baseline_clients AS (
   SELECT
-    DATE(submission_timestamp) AS submission_date,
-    COUNT(DISTINCT client_info.client_id) AS events_ping_distinct_client_count
+    DATE(
+      DATETIME(LEAST(ping_info.parsed_start_time, ping_info.parsed_end_time), 'UTC')
+    ) AS ping_date,
+    client_info.client_id,
+    normalized_channel AS channel,
+    normalized_country_code AS country
   FROM
-    fenix.events_unnested
+    fenix.baseline
+  WHERE
+    metrics.timespan.glean_baseline_duration.value > 0
+    AND LOWER(metadata.isp.name) <> "browserstack"
```

Comment:
This filter appears to filter out all data.
```suggestion
```

Reply:
At least this appears to be the case for older submission dates (e.g. `2020-01-01`)

Reply:
The active_users_aggregate begins to include mobile data starting from 2021-01-01. Let's set the backfill start date to be 2021-01-01 as well.

Reply:
Got it, thanks for confirming.